Jenkinsfile

```diff
@@ -17,6 +17,10 @@
 // I.e. for testing library changes
 //@Library(value='pipeline-lib@your_branch') _
 
+// Next version should be higher than the release or a very large number for master
+base_branch = 'release/2.6'
+next_version = '2.7'
+
```

Comment:
Why doesn't this work?
```
String next_version = base_branch
```

Reply:
It does, but you had requested more clarity with the next_version definition.